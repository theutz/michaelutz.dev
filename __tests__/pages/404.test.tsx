import { render, screen } from "@testing-library/react"
import NotFoundPage from "pages/404"

describe("<NotFoundPage />", () => {
  it("renders", () => {
    // Arrange

    // Act
    const { asFragment } = render(<NotFoundPage />)

    // Assert
    expect(asFragment()).toMatchSnapshot()
  })
})
