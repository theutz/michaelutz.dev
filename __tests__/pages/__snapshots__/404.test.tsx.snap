// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> renders 1`] = `
<DocumentFragment>
  <div
    class="bg-white min-h-screen px-4 py-16 sm:px-6 sm:py-24 md:grid md:place-items-center lg:px-8"
  >
    <div
      class="mx-auto max-w-max"
    >
      <main
        class="sm:flex"
      >
        <p
          class="text-4xl font-extrabold text-indigo-600 sm:text-5xl"
        >
          404
        </p>
        <div
          class="sm:ml-6"
        >
          <div
            class="sm:border-l sm:border-gray-200 sm:pl-6"
          >
            <h1
              class="text-4xl font-extrabold tracking-tight text-gray-900 sm:text-5xl"
            >
              Page not found
            </h1>
            <p
              class="mt-1 text-base text-gray-500"
            >
              Please check the URL in the address bar and try again.
            </p>
          </div>
          <div
            class="flex mt-10 space-x-3 sm:border-l sm:border-transparent sm:pl-6"
          >
            <a
              class="inline-flex items-center px-4 py-2 text-sm font-medium text-white bg-indigo-600 border border-transparent rounded-md shadow-sm hove:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
              href="#"
            >
              Go back home
            </a>
          </div>
        </div>
      </main>
    </div>
  </div>
</DocumentFragment>
`;
